spec/javascripts/app/views/comment_stream_view_spec.js

```diff
@@ -421,6 +421,17 @@ describe("app.views.CommentStream", function(){
       expect(submitCallback).not.toHaveBeenCalled();
     });
 
+    it("should not submit the form when cmd key is pressed", function(){
+      this.view.render();
+      var form = this.view.$("form");
+      form.submit(submitCallback);
+
+      var e = $.Event("keydown", { which: Keycodes.ENTER, metaKey: false });
+      this.view.keyDownOnCommentBox(e);
+
+      expect(submitCallback).not.toHaveBeenCalled();
+    });
```

Comment:
This should already be included in the test above. Maybe change its description to something like `shouldn't submit the form without the ctrl or cmd keys` and add `metaKey: false` when creating the event?